apps/token/src/components/vega-wallet/vega-wallet.tsx

```diff
@@ -180,15 +180,15 @@ const VegaWalletConnected = ({ vegaKeys }: VegaWalletConnectedProps) => {
         </div>
       ))}
       <WalletCardActions>
-        <Link style={{ flex: 1 }} to={Routes.GOVERNANCE}>
-          <span className="flex items-center justify-center w-full px-28 border h-28 bg-white text-black">
+        <Link className="flex-1 pr-8" to={Routes.GOVERNANCE}>
+          <Button variant={'secondary'} className="w-full">
```

Comment:
chore (blocking): same in here, `<a><button></button></a>` is not valid in terms of HTML semantics